src/app/qgsvectorlayerproperties.cpp

```diff
@@ -1470,6 +1472,33 @@ void QgsVectorLayerProperties::updateAuxiliaryStoragePage()
 
     // update actions
     mAuxiliaryLayerActionNew->setEnabled( false );
+
+    const QgsAuxiliaryLayer *alayer = mLayer->auxiliaryLayer();
+    if ( alayer )
+    {
+      const int fields = alayer->auxiliaryFields().count();
+      mAuxiliaryStorageFieldsLineEdit->setText( QString::number( fields ) );
+
+      // add fields
+      mAuxiliaryStorageFieldsTree->clear();
+      Q_FOREACH ( const QgsAuxiliaryField &field, alayer->auxiliaryFields() )
+      {
+        const QgsPropertyDefinition prop = field.propertyDefinition();
+        QTreeWidgetItem *item = new QTreeWidgetItem();
+
+        if ( prop.origin() == QgsPropertyDefinition::Pal )
+          item->setText( 0, "Pal" );
```

Comment:
This should be "labeling". PAL is an internal name for the label placement engine, it shouldn't be exposed anywhere to users

Reply:
done